codegen/field.gotpl

```diff
@@ -104,14 +104,12 @@
 		if err != nil {
 			return nil, err
 		}
+		if tmp == nil {
```

Comment:
:thinking: 
Right, because directives always use `interface{}` they can always be nil, and we don't care what the datatype is. Unlike everywhere else where we cant compare a string to nil without getting a compile-time error.

:+1: